src/main/java/ca/on/oicr/gsi/somnus/Server.java

```diff
@@ -240,6 +246,64 @@ protected void renderContent(XMLStreamWriter writer) throws XMLStreamException {
                   os.write(mapper.writeValueAsBytes(result));
                 }
               }
+
+              break;
+            case "GET":
+              try {
+                ArrayNode inhibitionDump = mapper.createArrayNode();
+                Instant now = Instant.now();
+                for (Inhibition i : inhibitions) {
+                  if (i.expirationTime().isAfter(now) && !i.awoken())
+                    inhibitionDump.add(mapper.convertValue(i, ObjectNode.class));
+                }
+                t.getResponseHeaders().set("Content-type", "application/json");
+                t.sendResponseHeaders(200, 0);
+                try (final OutputStream os = t.getResponseBody()) {
+                  os.write(mapper.writeValueAsBytes(inhibitionDump));
+                }
+              } catch (final Exception e) {
+                e.printStackTrace();
+                t.sendResponseHeaders(400, 0);
```

Comment:
Doesn't need fixed, but I don't think it's fair to blame the client for all exceptions... 😆